### PR TITLE
Remove multi-save on SaveMediaManager

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/66
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/134
https://github.com/open-orchestra/open-orchestra/pull/706
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/83

### CODE REVIEW

**Comment by nicolasThal** [comment]
I think that you forgot to suppress the test associated

File: Media/Manager/SaveMediaManager.php [hunk 1]
```diff
@@ -27,50 +27,24 @@ public function __construct($tmpDir, ThumbnailManager $thumbnailManager, Uploade
...
-    {
-        foreach ($medias as $media) {
-            $this->saveMedia($media);
-        }
-    }
-
-    /**
      * @param MediaInterface $media
      */
     public function saveMedia(MediaInterface $media)
```
Comment (nicolasThal) [inline Media/Manager/SaveMediaManager.php]: is it still used ?

**Comment by Thiblef** [comment]
:+1:

File: Media/Manager/SaveMediaManager.php (hunk 1 above)
Comment (itkg-ngilain) [inline Media/Manager/SaveMediaManager.php]: Yes it's still used in the MoveUploadedFileListener

File: MediaModelBundle/Manager/MediaManager.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace OpenOrchestra\MediaModelBundle\Manager;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use OpenOrchestra\Media\Repository\FolderRepositoryInterface;
+use OpenOrchestra\Media\Model\MediaFolderInterface;
```
Comment (itkg-alavieille) [inline MediaModelBundle/Manager/MediaManager.php]: remove this use

File: Media/Manager/SaveMediaManager.php [hunk 3]
```diff
@@ -19,62 +21,60 @@ class SaveMediaManager implements SaveMediaManagerInterface
...
     {
         if (null !== ($file = $media->getFile())) {
```
Comment (itkg-alavieille) [inline Media/Manager/SaveMediaManager.php]: the variable $file is now never used, you can remove it.

**Comment by itkg-ngilain** [comment]
@itkg-alavieille : update done

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**